INSTALLATION.md

```diff
@@ -179,20 +179,15 @@ If you want to get down to business, you can start your [Massalian journey](http
 #### Automatically
 > **_NOTE:_** If you're not a developer, it's better to use automatic installation
 
-Simply use copy/paste the cmd line below in your terminal. The installation process will start and a success message will be displayed once done.
-
-Note that you must execute this script as an administrator in your terminal. To do so:
-1. Open Windows search bar 
-2. Type "cmd" 
-3. Right click on the "Open command prompt" icon and click on "Execute as an administrator"
-
-Now, copy the cmd below and paste it in the terminal you just opened.
+Follow the link below and download the file named "ThyraApp_windows-amd64.exe" on your computer. Then: 
+1. Open it by double clicking
+2. The script starts and here is what it does:
+** If you have Thyra already installed on your computer, it install Thyra's icon tray. 
+![windows_icontray_V0](https://user-images.githubusercontent.com/109611779/212294116-05e1dd37-ed3f-4e3e-b034-b02d782bc4ee.png)
+** If nothing is installed on your computer yet, it will install both.
```

Comment:
It's not [the rendering](https://github.com/massalabs/thyra/blob/7b0218b8f38ae351872eb38df5f6a37278ac3823/INSTALLATION.md) you expect I think.

Reply:
Made some modifications, thanks :)